cli/cmd/inject.go

```diff
@@ -80,17 +81,17 @@ sub-folders, or coming from stdin.`,
 				return err
 			}
 
-			configs, err := options.fetchConfigsOrDefault()
+			values, err := options.fetchConfigsOrDefault()
 			if err != nil {
 				return err
 			}
 			overrideAnnotations := map[string]string{}
-			options.overrideConfigs(configs, overrideAnnotations)
+			options.overrideConfigs(values, overrideAnnotations)
```

Comment:
see https://github.com/linkerd/linkerd2/pull/5005/files#diff-0c198b8ec41cebf0e60ade048b268139R340

I think it makes sense here to simply return the annotation overrides rather than passing them in.